sdk/src/Core/Amazon.Util/AWSSDKUtils.cs

```diff
@@ -218,15 +240,16 @@ public static string GetExtension(string path)
                 else if (IsPathSeparator(ch))
                     break;
             }
+
             return string.Empty;
-        }
 
-        // Checks if the character is one \ / :
-        private static bool IsPathSeparator(char ch)
-        {
-            return (ch == '\\' ||
-                    ch == '/' ||
-                    ch == ':');
+            bool IsPathSeparator(char ch)
```

Comment:
Should this still be static?

Reply:
Good catch. Unfortunately, at the time I had looked at it this was limited by the `<LangVersion>` on .NET Framework (https://github.com/aws/aws-sdk-net/issues/3316). Static local functions required version 9.